### ISSUE TITLE
Cannot load module via CommonJS/Browserify.

### ISSUE THREAD

**Issue by RonanCodes** [krtnio/angular-pretty-xml#2]
When I try to load this module into my angular application via CommonJS/Browserify the module is not found.

In this form dependencies are loaded in like so:

``` js
var myModule = angular.module('app', [
    require('angular-pretty-xml').name,
]);
```

The dependency we are requiring has to have a wrapper that looks like so:

``` js
'use strict';
/*global module,require*/
var angular = require('angular');
require('angular-pretty-xml/src/angular-pretty-xml.js');
module.exports = angular.module('prettyXml');
```

The wrapper is defined in the `package.json`:

``` json
{
    "name": "app",
    "browser": {
        "angular-pretty-xml": "./app/client/wrappers/angular-pretty-xml.js"
    }
}
```

This approach works with every other module I've tried so far.
I noticed that the this module does not follow the same structure as others.
It doesn't have the same wrapper that is.

**Comment by RonanCodes** [comment]
Fixed issue, pull request #3

**Comment by v12** [comment]
Thanks for your contribution! I'll need to have a look into this issue because it should work with CommonJS. In fact, it uses [Universal Module Definition](https://github.com/umdjs/umd), which is compatible with CommonJS, RequireJS as well as browser globals if none of the module loaders is used.

I really like how detailed your suggestion is, however, it involves breaking changes and does break compatibility with other module loaders/bundlers (for instance, Webpack). The way you described works absolutely great when Browserify is used in the project but not every developer uses this amazing tool, preferring other things (there are a lot of nice tools, which are all about people's preferences 😃). Moreover, it does require modules to be "re-defined" in `package.json`. I know it's done only once but still it's an additional action. Basically, UMD was created to solve these issues, so libraries can be used hassle-free with different loaders/bundlers.

I doubt that the way the module is defined should be changed, however, I'm confident that your issue should be investigated in order to make it work as it's expected. Do you mind to provide some details of your current project configuration? Thanks!
